src/status_im2/contexts/quo_preview/tabs/segmented_tab.cljs

```diff
@@ -12,32 +13,41 @@
     :options [{:key   28
                :value "28"}
               {:key   20
-               :value "20"}]}])
+               :value "20"}]}
+   {:label "blur?:"
```

Comment:
The colon seems unnecessary after the question mark. Maybe just `Blur?` since the text is for the user (dev)?